pom.xml

```diff
@@ -191,6 +191,18 @@
 	</build>
 
 	<profiles>
+		<profile>
+			<!-- This profile prevents local maven builds from failing if uncommitted changes are present -->
+			<id>local-dev</id>
+			<activation>
+				<property>
+					<name>!env.CI</name>
```

Comment:
Is this property set on Jenkins?

Reply:
Yes, see https://github.com/jenkinsci/jenkins/pull/5370